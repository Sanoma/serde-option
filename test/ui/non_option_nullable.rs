use serde_option::serde_option;

#[serde_option]
struct Foo {
    #[nullable]
    x: u64,
}

fn main() {}
