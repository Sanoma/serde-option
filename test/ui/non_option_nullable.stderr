error: `#[nullable]` may only be used on fields of type `Option<T>`.
 --> test/ui/non_option_nullable.rs:6:5
  |
6 |     x: u64,
  |     ^
